### PR TITLE
[13.0][MIG] res_company_category

### PR DESCRIPTION
**Pull request by quentinDupont**

### CODE REVIEW

File: res_company_category/views/view_res_company_category.xml [hunk 1]
```diff
@@ -6,12 +6,12 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 <odoo>
 
-
     <record id="view_res_company_category_tree" model="ir.ui.view">
+        <field name="name">res.company.category.tree</field>
```
Comment (astirpe) [inline res_company_category/views/view_res_company_category.xml]: No need to add the name here. Odoo automatically does the job.

File: res_company_category/views/view_res_company_category.xml [hunk 2]
```diff
@@ -21,6 +21,7 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 
     <record id="view_res_company_category_form" model="ir.ui.view">
+        <field name="name">res.company.category.form</field>
```
Comment (astirpe) [inline res_company_category/views/view_res_company_category.xml]: No need to add the name here. Odoo automatically does the job.

**Review by astirpe — CHANGES_REQUESTED** [review]

File: res_company_category/views/view_res_company_category.xml (hunk 1 above)
Comment (quentinDupont) [reply res_company_category/views/view_res_company_category.xml]: I misread the doc ! changes made

**Review by astirpe — APPROVED** [review]
Code review OK (I didn't test it).

**Comment by quentinDupont** [comment]
I don't understand why travis fails ?
Thank for the review @astirpe !

**Comment by astirpe** [comment]
@quentinDupont could you rebase?

**Comment by quentinDupont** [comment]
> @quentinDupont could you rebase?

:tada:

**Pull request closed without merging**